codec/src/main/java/io/netty/handler/codec/string/LineSeparator.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2016 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.handler.codec.string;
+
+import io.netty.util.internal.ObjectUtil;
+
+public class LineSeparator {
+
+    public static final LineSeparator DEFAULT = new LineSeparator(System.getProperty("line.separator", "\n"));
+    public static final LineSeparator UNIX = new LineSeparator("\n");
+    public static final LineSeparator WINDOWS = new LineSeparator("\r\n");
+    public static final LineSeparator MAC = new LineSeparator("\r");
```

Comment:
Would it be a good idea to include this? I'm somewhat worried that people misunderstands what this really is.